chain/stmgr/call.go

```diff
@@ -201,6 +207,11 @@ func (sm *StateManager) CallWithGas(ctx context.Context, msg *types.Message, pri
 		)
 	}
 
+	filVested, err := sm.GetFilVested(ctx, ts.Height()+1)
```

Comment:
nit: store the epoch in a variable so we don't accidentally use `ts.Height()` in the somewhere in the future.